### PR TITLE
Docker Tools: Add SSL support section

### PR DESCRIPTION
**Pull request by ghogen**

### CODE REVIEW

File: docs/containers/debug-netcore.md [hunk 1]
```diff
@@ -31,4 +31,32 @@ MetaDescription: Debug a .NET Core app running in a Docker container, using Visu
 1. Optionally, set a breakpoint.
 1. Start debugging! (`kb(workbench.action.debug.start)`)
 
-For additional customization options, see the documentation on [Tasks](/docs/containers/reference.md) and [Debug containerized apps](/docs/containers/debug-common.md).
+## Running and debugging with SSL support
+
+If you are using SSL (using the HTTPS protocol), you will need to make a few changes to your configuration.
```
Comment (ucheNkadiCode) [inline docs/containers/debug-netcore.md:36]:
```suggestion
To enable SSL (using the HTTPS protocol), you will need to make a few changes to your configuration.
```

**Review by ucheNkadiCode — COMMENTED** [review]
This is great! Just one suggestion and I approve!

**Review by bwateratmsft — APPROVED** [review]
Agree with Uche's suggestion.

**Comment by ghogen** [comment]
@gregvanl  Ready to merge.

**Pull request merged**